### ISSUE TITLE
Unable to run binary

### ISSUE THREAD

**Issue by Cerberussian** [doitintl/kube-no-trouble#1]
Hi Stepan,

I downloaded Linux release and after unpacking and executing `./kubent` nothing happens.
`/chmod 777 kubent` did not help
executing `./kubent -h` also shows nothing

I have admin role in cluster and running
kube config is in default location
I'm using 18.04.4 LTS (Bionic Beaver)

**Comment by eranchetz** [comment]
@stepanstipl Looks like this was compiled in alpine and could not work on debian/fedora based 

```
# strace ./kubent -h
...
open("/lib/ld-musl-x86_64.so.1", O_RDONLY) = -1 ENOENT (No such file or directory)
...
```

@Cerberussian thanks for noticing we will work on fixing this soon

**Comment by stepanstipl** [comment]
Thanks @Cerberussian for reporting this, I was able to reproduce on Ubuntu, so I will try to fix soon.

@ebendutoit The go binary is compiled as static, and works if used directly. It seems it's the UPX Packer (https://upx.github.io/), which is used to compress the binary,  is the culprit here.

**Comment by stepanstipl** [comment]
This should be fixed in #2 and in the latest **0.2.0** release
 - https://github.com/doitintl/kube-no-trouble/releases/download/0.2.0/kubent-0.2.0-linux-amd64.tar.gz.

@Cerberussian please give it a go and let me know. 👍 

@eranchetz you were right, the `CGO_ENABLE` variable was not propagated in the GH actions build scenario... was a static binary, but only when built on my laptop 🤦
